### ISSUE TITLE
How to troubleshoot icons that aren't replacing the default? 

### ISSUE THREAD

**Issue by MikeTheGreat** [Donnnno/Arcticons#1767]
I created the 'Tello' icon, submitted a PR, the PR has been added to the main branch, that commit had been released through the FDroid store, and I can search for and find that icon.

However, the actual app is still showing the original icon (not the arcticon one)

I'm sure I've done something dumb somewhere - maybe I typed the info into appfilter.xml wrong, or I didn't find the right info to begin with 

Is there a guide for troubleshooting this?
I'd greatly appreciate any advice or tips about how to run this down.

**Comment by Donnnno** [comment]
I'd say, double-check the appfilter entry!

**Comment by wwwwwwari** [comment]
Its lines in the appfilter.xml are wrong. Right now the component info is split into two lines:


```
	<item component="ComponentInfo{ccom.tello.ui}" drawable="tello"/>
	<item component="ComponentInfo{com.keepcalling.ui.MainActivity}" drawable="tello"/>
```
	
What it should be:
```
	<item component="ComponentInfo{com.tello.ui/com.keepcalling.ui.MainActivity}" drawable="tello"/>
```

**Comment by MikeTheGreat** [comment]
My bad!
Sorry for the mistake and thank you for finding it!
I've put in a PR to fix that: #1768
